frontend/package.json

```diff
@@ -35,6 +37,7 @@
   "devDependencies": {
     "@graphql-codegen/cli": "^5.0.2",
     "@graphql-codegen/client-preset": "4.3.3",
+    "@types/dompurify": "^3.0.5",
```

Comment:
What are these types and why do you need it?

Reply:
I removed It